cybereason/help.md

```diff
@@ -0,0 +1,177 @@
+# Description
+
+The Cybereason platform provides military-grade cyber security with real-time awareness and detection.
+
+# Key Features
+
+* Search files on machines
+
+# Requirements
+
+* Requires an username and password
```

Comment:
```suggestion
* Requires a Cybereason username and password
* Cybereason account configured as shown [here](https://nest.cybereason.com/user/login?destination=/documentation/product-documentation/191/search-and-browse-files-machines-0#pre-requisites) for File Search capabilities
```

CC @jschipp-r7 - we talked about this in our email thread with Cybereason - any thoughts?